packer.json

```diff
@@ -0,0 +1,21 @@
+{
+  "builders": [{
+		"ami_name": "flugel-test",
+		"instance_type": "t2.micro",
+		"region": "us-east-1",
+		"type": "amazon-ebs",
+		"source_ami": "ami-039a49e70ea773ffc",
+		"ssh_username": "ubuntu"
+	}],
+	"provisioners": [{
```

Comment:
I don't see where are you installing traefik now with packer.

Reply:
For me is better to install Traefik along with the S3-Proxy at launch time. But if this a must I can have it installed in the image build.